src/NuGet.Services.AzureSearch/VersionList/SearchIndexChangeType.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch
+{
+    public enum SearchIndexChangeType
+    {
+        /// <summary>
+        /// The only change necessary is updating the version list. Some potentially no-op cases also fall into this
+        /// category to support reflowing.
+        /// </summary>
+        UpdateVersionList,
+
+        /// <summary>
+        /// The latest version has been deleted or unlisted, so we need to replace the latest version metadata with an
+        /// older version's metadata.
+        /// </summary>
+        DowngradeLatest,
+
+        /// <summary>
+        /// Update the existing latest version's metadata and version list.
+        /// </summary>
+        UpdateExisting,
```

Comment:
>UpdateExisting [](start = 8, length = 14)

What do you think of `UpdateLatest`?

Reply:
Yeah, I was torn between `UpdateLatest` and `UpdateExisting`. Whatever name we choose, this enum value is what we use for both the "reflow latest" and "change latest to later version" cases.

If this one makes more sense to you, I'll change it.